docs/learn.md

```diff
@@ -400,7 +400,318 @@ over the way these JavaScript packages are downloaded and bundled.
+single project with both OCaml native executables and frontend applications that
+are built with Melange, and even share code between both platforms in an easy
+manner.
+
+The way Dune and Melange work with each other is as follows: Dune orchestrates
+and plans the work needed to compile a project, copies files when needed, and
+prepares everything so that Melange takes OCaml source files and convert them
+into JavaScript code.
+
+Let’s now dive into the Melange compilation model and go through a brief guide
+on how to work with Dune in Melange projects.
+
+### Compilation model
+
+Melange compiles a single source file (either `.ml` or `.re` for
+[Reason](https://reasonml.github.io/en/) syntax) to a single JavaScript module.
+This simplifies debugging the produced JavaScript code and allows to import
+assets like CSS files and fonts in the same way as one would do in a JavaScript
+project. Even if Melange does not handle the bundling of JavaScript code for web
+applications, its compilation model allows integration with tools such as
+[Webpack](https://webpack.js.org/), or [other
+alternatives](https://npmtrends.com/@vercel/ncc-vs-esbuild-vs-parcel-vs-rollup).
+
+The [Melange opam template](https://github.com/melange-re/melange-opam-template)
+can be used as reference, as it provides an example of how to integrate Melange
+with Webpack.
+
+### Dune for Melange developers
+
+Dune is an OCaml build system that Melange projects can use to specify libraries
+and applications. It’s optimized for monorepos and makes project maintenance
+easier. This section provides an overview of Dune’s features and explains how to
+use it to build Melange applications.
+
+#### Features
+
+Dune is designed with OCaml in mind, which makes it an ideal tool for Melange
+developers. It provides several benefits, including:
+
+- Easy specification of libraries and executables.
+- Optimized for monorepos: no need for `npm link` or similar solutions.
+- Easy project maintenance, as one can rearrange folders without updating the
+  paths to libraries.
+- Hygiene is maintained in Dune by never writing in source folders by default,
+  unless explicitly configured to do so. All the compilation artifacts are
+  placed in a separate `_build` folder.
+- Dune provides a variety of additional features including [cram
+  tests](https://dune.readthedocs.io/en/stable/tests.html), integration with
+  [Odoc](https://dune.readthedocs.io/en/stable/documentation.html), Melange,
+  [Js_of_ocaml](https://dune.readthedocs.io/en/stable/jsoo.html), [watch
+  mode](https://dune.readthedocs.io/en/stable/usage.html#watch-mode), Merlin/LSP
+  integration for editor support, [cross
+  compilation](https://dune.readthedocs.io/en/stable/cross-compilation.html),
+  and [generation of `opam`
+  files](https://dune.readthedocs.io/en/stable/opam.html#generating-opam-files).
+
+#### Creating a new project
+
+To understand how to use Dune, let’s create a small Melange application using
```

Comment:
"using it" isn't adding anything to the sentence.

Reply:
https://github.com/melange-re/documentation-site/pull/25/commits/9e697f0447b4f924b17fed14e27de315a6a50bd6